apps/condo/domains/notification/schema/SyncRemoteClientService.js

```diff
@@ -48,10 +54,25 @@ const SyncRemoteClientService = new GQLCustomSchema('SyncRemoteClientService', {
                     }
                 }
 
+                /**
+                 * Clear already used pushTokenVoIP to avoid collisions
+                 */
+                if (pushTokenVoIP) {
+                    const presentRemoteClient = await getByCondition('RemoteClient', { pushTokenVoIP })
```

Comment:
Is it unique? Does remote client has a constraint?

Reply:
Yes, it's unique and we have constraint. but, some times, especially on dev environments, pushTokens use to migrate from device to device, and we had issues on that, so we reconnect pushToken to last remote client, who declared it.